src/node.js

```diff
@@ -206,6 +207,13 @@ class Node {
             this.api.apiHost = addr.address
             this.api.apiPort = addr.port
 
+            if (gwmatch) {
+              this.gatewayAddr = gwmatch[2]
+              const addr = multiaddr(this.gatewayAddr).nodeAddress()
```

Comment:
how is the a breaking change, currently the `gatewayAddr` doesn't exist from what I can tell?